documentation/src/main/wiki/HighLevelArchitecture.md

```diff
@@ -13,7 +13,7 @@ Diagram below depicts high level Knot.x architecture.
 
 Custom business logic can be encapsulated in dedicated Knots / Adapters.
 
-Knot is module which defines custom step while [request routing](#KnotRouting). It can process custom
+Knot is module which defines custom step while [[request routing|KnotRouting]]. It can process custom
```

Comment:
Awkward grammar.

_A Knot is a module which defines a custom step in the process of [[request routing|KnotRouting]]._